src/Microsoft.AspNet.SignalR.Client/Connection.cs

```diff
@@ -511,35 +511,40 @@ Task StartNegotiation()
                                 {
                                     VerifyProtocolVersion(negotiationResponse.ProtocolVersion);
 
-                                    if (negotiationResponse.ProtocolVersion.Equals("2.0") && !string.IsNullOrEmpty(negotiationResponse.RedirectUrl))
+                                    if (negotiationResponse.ProtocolVersion.Equals("2.0"))
```

Comment:
This looks unrelated...

Reply:
Yeah, the most recent commit is on the wrong branch.

Reply:
Fixed